Code/Framework/AzFramework/AzFramework/Viewport/CameraInput.cpp

```diff
@@ -213,9 +213,9 @@ namespace AzFramework
 
     Camera Cameras::StepCamera(const Camera& targetCamera, const ScreenVector& cursorDelta, const float scrollDelta, const float deltaTime)
     {
-        for (int i = 0; i < m_idleCameraInputs.size();)
+        for (int idleIndex = 0; idleIndex < m_idleCameraInputs.size();)
         {
-            auto& cameraInput = m_idleCameraInputs[i];
+            auto& cameraInput = m_idleCameraInputs[idleIndex];
             const bool canBegin = cameraInput->Beginning() &&
                 AZStd::all_of(m_activeCameraInputs.cbegin(), m_activeCameraInputs.cend(),
                               [](const auto& input)
```

Comment:
Nit: This part of the condition
````
(!cameraInput->Exclusive() || (cameraInput->Exclusive() && m_activeCameraInputs.empty()))
````
can be simplified to
````
(!cameraInput->Exclusive() || m_activeCameraInputs.empty())
````

Reply:
Ah good spot! You're quite right, will update 👍 Thanks!